### PR TITLE
docs: improve wording of file-extension-in-import docs

### PR DESCRIPTION
**Pull request by cherryblossom000**
https://github.com/mysticatea/eslint-plugin-node/pull/325

### CODE REVIEW

File: docs/rules/file-extension-in-import.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ import foo from "./path/to/a/file" // maybe it's resolved to 'file.js' or 'file.
 export * from "./path/to/a/file"
 ```
 
-However, [--experimental-modules](https://medium.com/@nodejs/announcing-a-new-experimental-modules-1be8d2d6c2ff) has declared to drop the file extension omition.
+However, when using [`--experimental-modules`](https://medium.com/@nodejs/announcing-a-new-experimental-modules-1be8d2d6c2ff), file extensions are required.
```
Comment (aladdin-add) [inline docs/rules/file-extension-in-import.md:14]: esm has been stable in node.js, so we may not need to mention the flag?

Comment (scagood) [reply docs/rules/file-extension-in-import.md:14]: Agreed, I think this entire line can be removed
```suggestion
```

**Review by scagood — APPROVED** [review]
Thank you!

**Review by aladdin-add — APPROVED** [review]
LGTM, thanks!

**Pull request merged**